swap_meet/clothing.py

```diff
@@ -1,2 +1,23 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    '''
+    Clothing class is the child class of Item class that inherits methods for condition of the clothing.
+    '''
+    
+    def __init__(self, condition = 0):
+        '''
+        Inherits Item methods.
+        '''
+        super().__init__(condition = condition, category = "Clothing")
```

Comment:
👍

Since you have the condition and category parameters in the correct order when you pass it to the parent's constructor, you can also write line 12 like:
`super().__init__(condition, "Clothing")`

Also good to see "Clothing" directly passed to the parent constructor so that we don't give users the chance to change the category for a Clothing instance to something else (at least not from the constructor).